extensions/amp-video/0.1/amp-video.js

```diff
@@ -686,6 +687,24 @@ export class AmpVideo extends AMP.BaseElement {
     listenOncePromise(this.video_, 'loadedmetadata').then(() =>
       this.onVideoLoaded_()
     );
+    this.setUpCaptions_();
+  }
+
+  /** Connects to amp-story-captions component. */
+  setUpCaptions_() {
+    const captionsId = this.element.getAttribute('captions-id');
+    if (!captionsId) {
+      return;
+    }
+    const captionsElement = document.getElementById(captionsId);
```

Comment:
- Plz use `this.win.document` instead
- Let's do `amp-story-captions#${escapeCssSelectorIdent(captionsId)}` just to be a bit safer about what we're getting, and ensure that it gets an element that has `getImpl` and then `setVideoElement` methods declared